markdown/1.x/en/core-result-dispatcher.md

```diff
@@ -20,7 +20,7 @@ The function supplied in `fn` is invoked with data for the subscribed
 
 #### All available topics
 * `result`
-* `campaigns` - special
+* `campaign` - special
```

Comment:
Did you check if there are ant additional occurences of `campaigns` in the context of topic name?

Reply:
Yes It's checked already

Reply:
OK.